README.md

```diff
@@ -1 +1,135 @@
-# Body-Building
\ No newline at end of file
+<a id="readme-top"></a>
+<div align="center">
+  <h1><b>SUPERPOWER FITNESS CENTER</b></h1>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+  - [Key Features](#key-features)
+- [🚀 Live Demo](#live-demo)
+- [🎥 Loom Video](#Loom-Video)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<br>
+
+# 📖 **SUPERPOWER FITNESS CENTER** <a id="about-project"></a>
+    **SUPERPOWER FITNESS CENTER** is a website that showcases the services and facilities offered by SUPERPOWER FITNESS CENTER, a gym that helps you achieve your fitness goals with personalized training and support. Whether you want to lose weight, gain muscle, or improve your health, this website will help you find the best plan for you.
+<br>
+
+## 🛠 **Built With** <a id="built-with"></a>
+- HTML
+- CSS
+- JavaScript
+<br>
+
+## **Key Features** <a id="key-features"></a>
+- **Ease of use**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+# 🚀 **Live Demo** <a id="live-demo"></a>
+- [Click Here](https://chawkimbarki.github.io/Body-Building/)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🎥 **Loom Video** <a id="Loom-Video"></a>
+- [Click Here](#)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ ] Great job for working on the deployement but I think the link is broken here sim=nce when I open the live demo so kindly make sure you have the right link 💯🔥✨to have it access online as required in the requirements 🎉

![image](https://user-images.githubusercontent.com/58553711/235010853-db69e8ff-14b6-4cff-816c-4dc0ebaa4046.png)